db/schema.rb

```diff
@@ -21,6 +21,15 @@
     t.index ["user_id"], name: "index_notes_on_user_id"
   end
 
+  create_table "social_accounts", force: :cascade do |t|
```

Comment:
Aren't we missing a foreign key definition here? The same change would be required to the migration, too, accordingly.